torch/csrc/cudnn/Conv.cpp

```diff
@@ -402,9 +542,10 @@ static void check_bias_size(THVoidTensor* bias, THVoidTensor* weight, int groups
   }
 }
 
-static void check_expected_output_size_is_valid(THVoidTensor* input, THVoidTensor* output, THVoidTensor* weight,
-                                                const std::vector<int>& pad, const std::vector<int>& stride,
-                                                const std::vector<int>& dilation)
+static void check_expected_output_size_is_valid(
+    THVoidTensor* input, THVoidTensor* output, THVoidTensor* weight,
+    const std::vector<int>& pad, const std::vector<int>& stride,
+    const std::vector<int>& dilation)
```

Comment:
uhhh let's not change the formatting unless it's really necessary... I can't see what was wrong with these lines and it makes the diff soo much larger than it has to be (which means the review takes so much longer, or it's easy to skip important changes)